policy/lib/refs.rego

```diff
@@ -41,6 +41,15 @@ task_ref(task) := i if {
 		"name": object.get(r, "name", ""),
 		"kind": lower(object.get(r, "kind", "task")),
 	}
+} else := i if {
+	# Handle inlined Task definitions
+	_ref(task) == {}
+	i := {
+		# The Task definition itself is inlined without a name. Use a special value here to
+		# distinguish from other reference types.
+		"name": "<INLINE>",
```

Comment:
Perhaps pull the `"<INLINE>"` into a constant?

Reply:
Good suggestion. I changed this to `<NAMELESS>` instead, but did throw that into a constant.